tests/ActiveDataProviderTest.php

```diff
@@ -30,12 +28,14 @@ public function testQuery()
     {
         $query = Item::find();
         $provider = new ActiveDataProvider([
+            'connectionClass' => 'yii\redis\Connection',
```

Comment:
Does that mean that for every case of working with `ActiveDataProvider` and connection user will have to specify `connectionClass` now?

Reply:
In case the `ActiveDataProvider` is used. We can introduce yii2-redis' own `ActiveDataProvider` where this `className` is provided but it's just shortcut.